docs/dev/libvirt/README.md

```diff
@@ -247,6 +247,31 @@ This step allows installer and users to resolve cluster-internal hostnames from
 
 3. Reload NetworkManager to pick up the `dns` configuration change: `sudo systemctl reload NetworkManager`
 
+### Setting the resolver on Fedora 34+ with systemd-resolved
+
+Fedora 34 switched to utilizing systemd-resolved and changes how DNS resolution works on a per-interface basis. 
+
+If you are running Fedora 34, you must indicate that you want your resolver to point to the local dnsmasq resolver on the `tt0` interface.
+
+The below example uses the default `tt0` interface, change the interface name if you are utilizing a custom interface/bridge instead.
+
+```sh
+resolvectl dns tt0 127.0.0.1
+```
```

Comment:
I wonder if on f33+, it would be workable to not setup dnsmasq at all on the host, and make use of https://github.com/openshift/installer/commit/a6ef6c7d0418ba81ecf6e88d9bcc6e65e5ac6ca7 instead (+ maybe some `resolvectl domain tt0 ~tt.testing` configuration?)

Reply:
I did not notice this section before. Thanks for pointing it out- I will test with this solution as this is cleaner than setting it up the way I've proposed.